### ISSUE TITLE
Would be nice if the test exclude list had a link to the relevant blocking issue

### ISSUE THREAD

**Issue by AndyAyersMS** [dotnet/llilc#534]
Otherwise when fixing a known issue it may be hard to know which tests to re-enable.
```
<ExcludeList Issue="512" Include="$(XunitTestBinBase)\JIT\Methodical\MDArray\DataTypes\bool.cmd"/>
```

**Comment by libengu** [comment]
PR 549 is created to handle this issue.

**Comment by AndyAyersMS** [comment]
Fixed via #549.
